src/content/operations/cleanup/_index.en.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Uninstalling Submariner"
+date: 2020-12-23T21:25:11+01:00
+weight: 50
+---
+
+To properly uninstall Submariner from a cluster, follow the steps below:
+
+1. Delete Submariner related namespaces
+
+   On each participating cluster, issue the following command:
+
+   ```bash
+   kubectl delete namespace submariner-operator
+   ```
+
+   On the Broker cluster, issue the following command:
+
+   ```bash
+   kubectl delete namespace submariner-k8s-broker
+    ```
+
+2. Delete the Submariner CRDs
+
+   On each participating cluster, issue the following command:
+
+   ```bash
+   for CRD in `kubectl get crds | grep -iE 'submariner|multicluster.x-k8s.io'| awk '{print $1}'`; do kubectl delete crd $CRD; done
+   ```
+
+3. Delete ClusterRoles and ClusterRoleBindings
+
+   On each participating cluster, issue the following command:
+
+   ```bash
+   roles="submariner-operator submariner-operator-globalnet submariner-lighthouse submariner-networkplugin-syncer"
+   kubectl delete clusterrole,clusterrolebinding $roles --ignore-not-found
+   ```
+
+4. Remove Submariner's iptables chains
+
+   On all nodes in each participating cluster, issue the following commands:
+
+   ```bash
+   iptables --flush SUBMARINER-INPUT
+   iptables -D INPUT $(iptables -L INPUT --line-numbers | grep SUBMARINER-INPUT | awk '{print $1}')
+   iptables --delete-chain SUBMARINER-INPUT
+
+   iptables --flush SUBMARINER-POSTROUTING
+   iptables -D INPUT $(iptables -L INPUT --line-numbers | grep SUBMARINER-POSTROUTING | awk '{print $1}')
+   iptables --delete-chain SUBMARINER-POSTROUTING
+   ```
+
+   If Globalnet is enabled in the setup, issue the following commands as well
+
+   ```bash
+   iptables --flush SUBMARINER-GN-INGRESS
+   iptables -D INPUT $(iptables -L INPUT --line-numbers | grep SUBMARINER-GN-INGRESS | awk '{print $1}')
+   iptables --delete-chain SUBMARINER-GN-INGRESS
+
+   iptables --flush SUBMARINER-GN-EGRESS
+   iptables -D INPUT $(iptables -L INPUT --line-numbers | grep SUBMARINER-GN-EGRESS | awk '{print $1}')
+   iptables --delete-chain SUBMARINER-GN-EGRESS
+
+   iptables --flush SUBMARINER-GN-MARK
+   iptables -D INPUT $(iptables -L INPUT --line-numbers | grep SUBMARINER-GN-MARK | awk '{print $1}')
+   iptables --delete-chain SUBMARINER-GN-MARK
+   ```
+
+   {{% notice note %}}
+   Note that Kind based setup will have just `SUBMARINER-INPUT` chain.
```

Comment:
@Jaanki I'm not sure if this statement (i.e., Note that Kind based setup will have just `SUBMARINER-INPUT` chain) is valid. Can you please share additional info on why KIND based setup will have just SUBMARINER-INPUT chain and not others?

Reply:
@sridhargaddam I am not seeing other chains in AWS setup also. Not even on gateway nodes. Just SUBMARINER-INPUT chain.

Reply:
Probably you are looking at the wrong place. Please ping me when your setup is available and I'll walk-through how to look at various chains.

Reply:
ohk so the iptables are programmed inside the globalnet pod. I was looking on the nodes.

Reply:
They are programmed on the node itself. All Submariner Pods (i.e., Globalnet, Gateway and RouteAgent) run with HostNetworking set to true. So when you exec to the pods and check the iptables configuration, you are actually looking at the configuration on the nodes itself.